src/CloudGraphsDFG/services/CloudGraphsDFG.jl

```diff
@@ -192,7 +196,16 @@ function getFactor(dfg::CloudGraphsDFG, label::Union{Symbol, String})::DFGFactor
         error("Unable to retrieve the ID for factor '$label'. Please check your connection to the database and that the factor exists.")
     end
 
-    return getFactor(dfg, nodeId)
+    props = getnodeproperties(dfg.neo4jInstance.graph, nodeId)
+    factor = unpackFactor(dfg, props)
+
+    # Lastly, rebuild the metadata
+    factor = dfg.rebuildFactorMetadata!(dfg, factor)
+    # GUARANTEED never to bite us in the future...
+    # ... TODO: refactor if changed: https://github.com/JuliaRobotics/IncrementalInference.jl/issues/350
+    getSolverData(factor).fncargvID = factor._variableOrderSymbols
```

Comment:
just note fncargvID will be deprecated in v0.7 in favor of only having it in one place, ie. `factor._variableOrderSymbols` with getter `getVariableOrder`

Reply:
Great! fncargvID is a duplicate, would be good to remove.